src/app/content/highlights/components/ShowMyHighlightsStyles.tsx

```diff
@@ -44,6 +65,10 @@ export const HighlightSection = styled.div`
   display: flex;
   align-items: center;
   font-weight: bold;
+
+  > span {
```

Comment:
since we are using `dangerouslySetInnerHTML` we have additional spans with `.os-number`, `.os-divider` and `.os-text` classes. I'll list it like this:
```
 > .os-number,
 > .os-divider,
 > .os-text {
    overflow: hidden;
 }
```

Reply:
thank you, i had assumed you were only targeting the os-text, so if its important that you hit all of them its much nicer to have it explicitly written out